### ISSUE TITLE
iOS Crash: App crashing after answer question of reCaptcha

### ISSUE THREAD

**Issue by Vadim7394** [plaid/react-native-plaid-link-sdk#165]
Error:
```
Fatal Exception: NSInvalidArgumentException
-[PLKLinkAPIv2 requestAuthenticateWithRecaptcha:credentials:flexibleResponses:institution:publicToken:configuration:completion:]: unrecognized selector sent to instance 0x600003fd8000
objc_exception_throw
```
Stacktrace:
```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x7fff23e3cefe (Missing)
1  libobjc.A.dylib                0x7fff50ba89b2 objc_exception_throw
2  CoreFoundation                 0x7fff23e5dc34 (Missing)
3  CoreFoundation                 0x7fff23e4190c (Missing)
4  CoreFoundation                 0x7fff23e43bf8 (Missing)
5  LinkKit                        0x10838f0c7 (Missing)
6  LinkKit                        0x10833d321 (Missing)
7  LinkKit                        0x10830e4e2 (Missing)
8  WebKit                         0x7fff2d411f1d (Missing)
9  WebKit                         0x7fff2d5be863 (Missing)
10 WebKit                         0x7fff2d7c8de8 (Missing)
11 WebKit                         0x7fff2d27bb4a (Missing)
12 WebKit                         0x7fff2d4f7bf8 (Missing)
13 WebKit                         0x7fff2d26b00c (Missing)
14 WebKit                         0x7fff2d26a641 (Missing)
15 JavaScriptCore                 0x7fff26a7bdb5 (Missing)
16 JavaScriptCore                 0x7fff26a7bf22 (Missing)
17 CoreFoundation                 0x7fff23da0d31 (Missing)
18 CoreFoundation                 0x7fff23da0c5c (Missing)
19 CoreFoundation                 0x7fff23da0434 (Missing)
20 CoreFoundation                 0x7fff23d9b02e (Missing)
21 CoreFoundation                 0x7fff23d9a944 (Missing)
22 GraphicsServices               0x7fff38ba6c1a (Missing)
23 UIKitCore                      0x7fff48c8b9ec (Missing)
24 NexMobile                      0x1051d89f0 main + 7 (main.m:7)
25 libdyld.dylib                  0x7fff51a231fd (Missing)
26 ???                            0x1 (Missing)
```

**Comment by afh** [comment]
Hello @Vadim7394, which version of LinkKit (e.g. the Plaid CocoaPod) is being used?
An issue regarding crashes during recaptcha was addressed in [LinkKit version 1.1.33](https://github.com/plaid/plaid-link-ios/releases/tag/ios%2F1.1.33)

**Comment by Vadim7394** [comment]
Hello @afh, all work after update, thanks!
